FordLabs/PeopleMover
# 200 person tag update counter

## Issue
Resolves 200

## What was done
- [x] Updated the counter to reflect when person-tags are selected

## How to test
1.Create some person and add some person tags to them. Then select the person tag in the filter drop down and see the count adjust
